scripts/tutorials/wdl/README.md

```diff
@@ -0,0 +1,31 @@
+### Easily find information in each tutorial
+---
+
+The WDL scripts in this folder are designed to teach you aspects and features of WDL, using GATK as it's main tool. They follow the tutorial documents located on the WDL website, under the Tutorials category
+
+- https://software.broadinstitute.org/wdl/userguide/topic?name=wdl-tutorials
+
+Each  tutorial script located in this folder will have a link in its header that points to its specific tutorial document. You can copy and paste that link into your browser window and follow along with the document in order to write your own script. These are designed in a way that you should be using the scripts uploaded to this folder as a means of checking your work. 
+
+### Example data
+---
+
+Each tutorial document will link you to the specific `.zip` file associated with each script. These `.zip` files contain all the example data you will need to run the script, in addition to the sample script itself. You can find these `.zip` files here:
+
+- https://drive.google.com/drive/u/0/folders/0BwTg3aXzGxEDMUs0cURPcnFiQzA
```

Comment:
Are you sure this is a shareable link? Seems formatted specifically for people who have a Google Drive account. Make sure you use the "Get shareable link" button.